buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/PerformanceTest.java

```diff
@@ -43,7 +43,10 @@
  */
 @CacheableTask
 public class PerformanceTest extends DistributionTest {
-    private Property<String> baselinesProperty;
+    private static final String DEFAULT_BASELINE = "defaults";
+    private static final String FORCE_DEFAULT_BASELINE = "force-defaults";
+    private Property<String> configuredBaselines = getProject().getObjects().property(String.class).value(DEFAULT_BASELINE);
```

Comment:
Why aren't these properties `ListProperty`s? They are configured to a list of strings, right?

Reply:
We don't have to make that change right now, it may complicate things.